sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/OffHeapColumnVector.java

```diff
@@ -221,6 +221,13 @@ protected UTF8String getBytesAsUTF8String(int rowId, int count) {
     return UTF8String.fromAddress(null, data + rowId, count);
   }
 
+  @Override
+  public ByteBuffer getBytesUnsafe(int rowId, int count) {
```

Comment:
Can we replace it with:
```java
return ByteBuffer.wrap(getBytes(rowId, count));
```

Reply:
It seems fine to use `getBytes` here also - the function call will be inlined by JIT if this is in a hot path.